src/DataTable/DataTable.tsx

```diff
@@ -162,12 +171,25 @@ function DataTable<T>(props: TableProps<T>): JSX.Element {
 	const currentTheme = React.useMemo(() => createStyles(customStyles, theme), [customStyles, theme]);
 	const wrapperProps = React.useMemo(() => ({ ...(direction !== 'auto' && { dir: direction }) }), [direction]);
 
+	// const flages = React.useMemo(() => ({ ...(direction !== 'auto' && { dir: direction }) }), [direction]);
```

Comment:
Commented code should not be pushed to GitHub.

Reply:
Commented codes are removed. Thank You.